DiscordBotApis/Startup.cs

```diff
@@ -28,7 +29,14 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
 
-            services.AddSingleton<IGuildRepository, MockGuildRepository>();
+            //services.AddSingleton<IGuildRepository, MockGuildRepository>();
```

Comment:
Please don't leave any commented out code in commits.

Reply:
fixed